.github/workflows-source/release-testing.yml

```diff
@@ -182,10 +193,37 @@ jobs:
           cd ci/src/dependencies/
           $SHELL_WRAPPER python3 job/bazel_rust_ic_scanner_release_job.py
 
-  trigger-qualifier-workflow:
-    name: Qualify
-    needs: [ci-main, bazel-system-test-nightly, bazel-system-test-staging, bazel-system-test-hotfix, dependency-scan-release-cut]
-    uses: dfinity/dre/.github/workflows/qualify.yaml@main
-    secrets: inherit
-    with:
-      version: ${{ github.sha }}
+  setup-guest-os-qualification:
+    <<: *dind-small-setup
```

Comment:
Why do you think `dind-small-setup` is good fit for this CI job? Could you add a name to this CI job so it's consistent (title case)?

Reply:
Well the test is run on the farm, so the runner only needs to ping the farm and log the comments, no need to waste resources on that right? As for the of the CI job being consistent I didn't really understand the request. Can you give me an example?

Reply:
I would start with running this on dind-large like we do for all other system-tests. I don't think you have to worry about resource usage here since this workflow is only triggered on pushes to hotfix and rc branches which probably doesn't happen too often.